### DISCUSSION TITLE
Hello，我是来自中国的Flash开发者，我也想为AIR做出一些贡献……

### DISCUSSION THREAD

**Discussion by TKCB** [airsdk/Adobe-Runtime-Support#2173] [General]
你们好，我是TKCB-GO。

在中国之外，相信还有很多热爱Flash AS AIR技术的朋友。
但在中国，据我所知，仍在热爱和坚守Flash编程技术的，只有我们11RIA 闪客社区，以及我们的QQ群，以及我自己（TKCB-GO）。

有朋友给我们说了 Harman AIR 社区正在重建的事情。
我也想为Flash AIR共享一份你自己的力量，我这边其实也有很多Flash AS AIR的资料，当然大多数是中文的。

这是我的论坛，感兴趣的话，可以来看看：http://www.11ria.com/  （论坛里面有很多SWF内容，需要使用带有Flash Player的浏览器查看）
如果 Harman AIR 需要，论坛的大部分资料我们都可以开放下载，给你们足够的积分，但前提是确定是Harman官方的人员联系我们，因为很多技术资料都是我们无数人努力的心血。

这是我个人的网站，感兴趣的话，可以来看看：http://www.tkcb.cc/

这是我们修改、优化的中文的，AcitonScript 3.0 在线API文档：http://www.tkcb.cc/aAS3API/actionscript/3/

这是我自己在11RIA 闪客社区分享的AS3小练习：http://www.11ria.com/forum.php?mod=collection&action=view&ctid=7 （需要使用带有Flash Player的浏览器查看）

最后：如果Harman AIR需要一些关于Flash的技术资料（我大多数是中文的），那么可以技术人员与我们联系，我会尽可能的提供一些我所能提供的帮助。

上面就是我的毛遂自荐，希望没有打扰到你们。
感谢Macromedia公司、Adobe公司，Flash技术我热爱它。

另外，之前有网友建议我把我的文字翻译为英文，然后在Harman社区进行交流。
但我觉得没必要，因为现在谷歌翻译很强大了。
而且我是一个中国人，我其实英语非常不好，单词认识也就一两百个编程会用到的，口语也只会几句而已。那么为什么不用自己最擅长的语言呢？
我也希望大家都用自己的母语交流，毕竟谷歌等翻译工具，挺好的，基本意思不会有偏差。

**Comment by ajwfrost** [comment]
Thanks for this! I assume you've also come across the "Flash Center" from 2144, and https://www.flash.cn/?

We have the updated AS3 documentation available now under https://airsdk.dev/reference/actionscript/3.0/ but sadly don't have any translations. I suspect currently it will just be necessary to use something like Google Translate...

thanks

**Comment by TKCB** [comment]
是的，我看到了  https://airsdk.dev/reference/actionscript/3.0/   这个在线文档，确实没有翻译。
而且目前Adobe公司提供的AS3在线文档，最新版也只有英文，并没有中国的中文版本。

所以我们中国的Flash开发者，才将Adobe的FP30版本的中文本，下载下来，并做了细节优化，以便我们可以长久保留下来这份珍贵的技术资料。
我很喜欢我们给AS3在线文档制作的封面图，那是一个英雄（侠客）不屈服于世界（江湖）的意思，是我对Flash的热爱的坚守。
http://www.tkcb.cc/aAS3API/actionscript/3/

其实我们遇到的不仅仅是"Flash Center"。

我们经历了，中国最大的Flash技术论坛（9RIA天地会：www.9ria.com  网站已经不是原来的网站了）的关站过程，我们曾经试图过，努力去拯救这份资料。
这是我曾经试图拯救9RIA资料的一点聊天记录：http://www.11ria.com/forum.php?mod=viewthread&amp;tid=390

这里有我们抢救下来的9RIA的技术资料，需要的可以自己注册，进行下载：http://www.11ria.com/forum.php?mod=viewthread&amp;tid=40

这十几年，我们经历了很多，所以我作为中国最后的闪客（Flash AS）领头人，我希望为Flash技术的世界共享自己的一份力量。

2017年之后，我创立了11RIA 闪客社区（ www.11ria.com  ），我试图让中国的Flash技术开发者再聚集起来，可惜的是，热情早已不在，树倒猢狲散（很多技术高手都已经转型其他语言开发了）。

但我的热情并未磨灭。只是很多事情，我在做。但是有些事情，我不在做了。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "airsdk/Adobe-Runtime-Support"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月29日(星期四) 下午3:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [airsdk/Adobe-Runtime-Support] 我是来自中国的Flash开发者，TKCB-GO，我也想为AIR做出一些贡献…… (Discussion #2173)





 
Thanks for this! I assume you've also come across the "Flash Center" from 2144, and https://www.flash.cn/?
 
We have the updated AS3 documentation available now under https://airsdk.dev/reference/actionscript/3.0/ but sadly don't have any translations. I suspect currently it will just be necessary to use something like Google Translate...
 
thanks

**Reply by ajwfrost** [reply]
Interesting.. that seems a shame. I thought actually that the Flash development community was a bit more active in China since this is the only region in which Adobe (through 2144) still support the Flash Player browser plug-in.. we have a fair number of AIR licensees in China as well...

**Comment by TKCB** [comment]
是的，这很有意思，我也为曾经的自己做过的事情自豪。

中国的Flash社区，在曾经是最大的，至少当年的9RIA社区的创始人（黑羽）是这样宣传的。

在中国当年有几十万的开发者，当然很多人都是新手，或者只是其他语言的程序员，只是因为当时的环境潮流，然后顺带学了下Flash AS技术而已。

中国现在确实有很多Flash 从业者，但可惜的是，其实大部分人如果想换工作的话，已经很难找到Flash的工作了。（至少我是这样的）

但有一点，我是明确的，中国太多人，很聪明，他们会使用Flash技术，甚至自己技术很好，但不愿意分享，或者说现在不愿意分享了，当然的是很多人已经转型其他开发语言了，甚至不再做编程行业了（这点我很清楚）~~

所以这些年，Flash社区论坛，越来越少了，愿意交流FLASH技术的越来越少了……

但我努力让我们中国少有的，还在用或者关心或者对Flash有情怀的朋友，聚在一起，所以才有了11RAI和我们的QQ群（2011创建，到现在至少还保持活跃中）

看到Harman在重建AIR，我也挺喜欢或者说高兴，因为谁也不喜欢一个人孤单的战斗。

所以如果Harman需要我们中国的Flash开发者们的协助，请联系我，因为我们这边其他人好像并不关心这个事情。

希望一起加油！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "airsdk/Adobe-Runtime-Support"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月29日(星期四) 晚上6:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [airsdk/Adobe-Runtime-Support] Hello，我是来自中国的Flash开发者，我也想为AIR做出一些贡献…… (Discussion #2173)





 
Interesting.. that seems a shame. I thought actually that the Flash development community was a bit more active in China since this is the only region in which Adobe (through 2144) still support the Flash Player browser plug-in.. we have a fair number of AIR licensees in China as well...

**Reply by ajwfrost** [reply]
Thank you, it's lovely to hear your enthusiasm! Hopefully as we slowly improve the tools and APIs, it will become more attractive for people to use again..

**Comment by TKCB** [comment]
好的，如有需要请联系。
我也很高兴看到Flash技术再次生根发芽，老树新芽。
另外，以我对我们行业内的观察。其实很多时候，是缺少成体系的基础教程，特别是适合新手或者其他语言开发者学习的，很简单又学起来有效果的教程、案例、demo等等。
所以建议在这方面可以多做一些工作。因为我觉得对于新手和其他语言开发者，他们需要快速认识到Flash相关技术的优势，以及快速简单的教程引导他们，能上手学习和开发Flash/AIR。
谢谢。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "airsdk/Adobe-Runtime-Support"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月30日(星期五) 晚上6:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [airsdk/Adobe-Runtime-Support] Hello，我是来自中国的Flash开发者，我也想为AIR做出一些贡献…… (Discussion #2173)





 
Thank you, it's lovely to hear your enthusiasm! Hopefully as we slowly improve the tools and APIs, it will become more attractive for people to use again..

**Reply by ajwfrost** [reply]
Yes I agree on that, we definitely need to get more tutorials up and improve the "how to start using it" sort of process/documentation..

**Comment by TKCB** [comment]
嗯，是的。


祝福，Flash/AIR/AS技术，再次获得众生之力，老树新芽。


幸运与我们同在……




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "airsdk/Adobe-Runtime-Support"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月1日(星期六) 下午2:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [airsdk/Adobe-Runtime-Support] Hello，我是来自中国的Flash开发者，我也想为AIR做出一些贡献…… (Discussion #2173)





 
Yes I agree on that, we definitely need to get more tutorials up and improve the "how to start using it" sort of process/documentation..
